docs/lazy-adr/adr-006-da-interface.md

```diff
@@ -54,40 +55,40 @@ type StatusCode uint64
 
 // Data Availability return codes.
 const (
-	StatusUnknown StatusCode = iota
-	StatusSuccess
-	StatusTimeout
-	StatusError
+ StatusUnknown StatusCode = iota
```

Comment:
The one issue with this linter is it doesn't play well with Go code since Go code uses tabs (:crab:). Can you file an issue to follow-up with configuring the linter to allow tabs for these code blocks?

Reply:
Thanks for pointing out. Fixed already.